src/shared/components/oncoprint/TooltipUtils.ts

```diff
@@ -128,14 +131,27 @@ export function makeGeneticTrackTooltip_getCoverageInformation(
     let dispProfiledInMap:{[molecularProfileId:string]:string} = {};
     let dispNotProfiledIn:string[]|undefined = undefined;
     let dispNotProfiledGenePanelIds:string[] = [];
+    let profiledInTypes:{[type:string]:string}|undefined = undefined;
     if (profiled_in) {
         dispProfiledGenePanelIds = _.uniq((profiled_in.map(x=>x.genePanelId) as (string|undefined)[]).filter(x=>!!x) as string[]);
         dispProfiledIn = _.uniq(profiled_in.map(x=>x.molecularProfileId));
+        if (molecularProfileIdToMolecularProfile) {
+            profiledInTypes = _.keyBy(dispProfiledIn, molecularProfileId=>molecularProfileIdToMolecularProfile[molecularProfileId].molecularAlterationType);
+        }
         dispProfiledInMap = _.keyBy(dispProfiledIn);
         dispProfiledGenePanelIdsMap = _.keyBy(dispProfiledGenePanelIds);
     }
     if (not_profiled_in) {
         dispNotProfiledIn = _.uniq(not_profiled_in.map(x=>x.molecularProfileId)).filter(x=>!dispProfiledInMap[x]); // filter out profiles in profiled_in to avoid confusing tooltip (this occurs e.g. w multiple samples, one profiled one not)
+        if (profiledInTypes && alterationTypesInQuery && molecularProfileIdToMolecularProfile) {
+            let notProfiledInTypes = _.keyBy(dispNotProfiledIn, molecularProfileId=>molecularProfileIdToMolecularProfile[molecularProfileId].molecularAlterationType);
+            // add an entry to 'not profiled in' for each alteration type in the query iff the sample is not profiled in a profile of that type, and that type is not already accounted for.
+            // This is for the case of multiple study query - eg one study has CNA profile, the other doesnt, and we want to show in a tooltip from the other study that
+            //      the sample is not profiled for CNA. If the study actually has a CNA profile, then we wont show "not profiled for copy number alterations" because
+            //      that will be filtered out below because its in profiledInTypes or notProfiledInTypes. Otherwise, CNA will be in alterationTypesInQuery,
+            //      and it wont be covered in profiledInTypes or notProfiledInTypes, so it will make sense to say "copy number alterations" in that generality.
```

Comment:
This solution is a bit hacky. @alisman does the comment text make sense/make this more understandable?